src/gui/guiEngine.cpp

```diff
@@ -612,3 +612,44 @@ unsigned int GUIEngine::queueAsync(const std::string &serialized_func,
 	return m_script->queueAsync(serialized_func, serialized_params);
 }
 
+/******************************************************************************/
+bool GUIEngine::CheckAbsPath(const std::string &path, bool *write_allowed)
```

Comment:
We use lowerCamelCase for function names. :)

Reply:
It's a static function

Reply:
Aren't all functions named in lowerCamelCase? :thinking: